aksel.nav.no/website/pages/eksempler/popover/no-flip.tsx

```diff
@@ -38,6 +38,7 @@ export default withDsExample(Example);
 /* Storybook story */
 export const Demo = {
   render: Example,
+  desc: "Ved å sette `flip` til `false` tvinger du popoveren til å respektere valgt `placement` selv om det ikke er plass.",
```

Comment:
Could consider just removing the story tbh since it's not used
https://metabase.ansatt.nav.no/question/2711-radata-fra-component-usage?component=Popover&prop=flip&major=

Reply:
Good point